net-mail/offlineimap/offlineimap-7.3.3.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# Normally you need only one version of this.
+DISTUTILS_SINGLE_IMPL=1
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="threads,sqlite,ssl?"
+
+inherit distutils-r1
+
+DESCRIPTION="Powerful IMAP/Maildir synchronization and reader support"
+HOMEPAGE="https://www.offlineimap.org/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Nooo don't do that. It's easier to copypaste the URL to visit their Github page when it's in plaintext. Not like it's going to change.

Reply:
Oops. I thought it was alright for SRC_URI!

Reply:
Just a preference for proxy-maint since we have to open these Github pages all the time. No official policy about it.